.readthedocs.yml

```diff
@@ -1,20 +1,24 @@
 version: 2
 
 build:
-    image: latest
+  os: ubuntu-20.04
+  tools:
+    python: mambaforge-4.10
```

Comment:
See the associated RTD documentation [Making builds faster with mamba](https://docs.readthedocs.io/en/stable/guides/conda.html#making-builds-faster-with-mamba)... certainly seems to deliver :+1:

Reply:
Looks like the RTD docs have been updated since I last looked at it.  Speedier build is welcome.